### ISSUE TITLE
Nannou issues on X11

### ISSUE THREAD

**Issue by Caleb-o** [SuperCuber/facto_rs#1]
Trying to run the code:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: NoAvailableAdapter', src/main.rs:18:70
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

It is apparently an issue with X11 and several people have stated they cannot use it, even after installing the packages needed for their OS. Even when going through their [setup guide](https://www.guide.nannou.cc/getting_started/platform-specific_setup.html), all the packages are already installed.

Related issues for Nannou: [Closed and unsolved #653](https://github.com/nannou-org/nannou/issues/653)

I will come back if I can find a work-around or some way to solve it properly.

**Comment by Caleb-o** [comment]
After searching through many threads etc about nannou and vulkan, I found that Vulkan was eventually the issue that stopped things from running. I ran `vulkaninfo` for it to throw me an error too, which didn't give me much either. I ran `ls /usr/share/vulkan/icd.d/` which listed multiple json files with intel_^, amd_^ and nvidia_^ in their name. Since I have nvidia, I had to remove the other JSON files that weren't nvidia related, then vulkaninfo worked, as well as running the project after. All that to eventually go down a vulkan rabbit hole to find the issue, which was not fun.

It is now working, damn you vulkan!
